### PR TITLE
sqs queue implemented

### PR DESCRIPTION
**Pull request by kabolov**
Hi! Everything is done except tests (mark 5/6) my nickname in Discord is ossset(@kabolov), feel free to reach me out if you have any questions!

### CODE REVIEW

File: import-service/src/catalogBatchProcess.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      );
+
+      const newId = rows[0].id;
+
+      await client.query(
+        `insert into stocks (product_id, count) values ($1, $2)`,
+        [newId, count]
+      );
+
+      await client.query(
```
Comment (kharashun) [inline import-service/src/catalogBatchProcess.js:42]: Not a bug. This query might be omitted by combining responses from two queries above.
As another option, you could use WITH statement from Postgres. So there might be only one request.

File: import-service/src/catalogBatchProcess.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+const { Client } = require("pg");
```
Comment (kharashun) [inline import-service/src/catalogBatchProcess.js:1]: catalogBatchProcess might be created in Product service, as works with products and database.

File: import-service/serverless.yml [hunk 3]
```diff
@@ -8,6 +8,17 @@ provider:
...
+  environment:
+    SQS_URL:
+      Ref: SQSQueue
+    SNS_ARN:
+      Ref: SNSTopic
+    PG_HOST: lesson4-instance.c3zxepihdznj.eu-west-1.rds.amazonaws.com
+    PG_PORT: 5432
+    PG_DATABASE: lesson4
+    PG_USERNAME: postgres
+    PG_PASSWORD: wwIJ7CchntfGgnOsOzxk
```
Comment (kharashun) [inline import-service/serverless.yml:20]: hardcoded credentials - should be hidden - for example in .env file

File: import-service/serverless.yml [hunk 4]
```diff
@@ -17,6 +28,42 @@ provider:
...
+      Type: AWS::SNS::Subscription
+      Properties:
+        Endpoint: alexander.kabolov@gmail.com
+        Protocol: email
+        TopicArn:
+          Ref: SNSTopic
+    SNSSubscription2:
+      Type: AWS::SNS::Subscription
+      Properties:
+        Endpoint: aleksandr.kabolov@gmail.com
```
Comment (kharashun) [inline import-service/serverless.yml:60]: there could be different email

File: import-service/serverless.yml [hunk 5]
```diff
@@ -17,6 +28,42 @@ provider:
...
+      Type: AWS::SNS::Topic
+      Properties:
+        TopicName: createProductTopic
+    SNSSubscription:
+      Type: AWS::SNS::Subscription
+      Properties:
+        Endpoint: alexander.kabolov@gmail.com
+        Protocol: email
+        TopicArn:
+          Ref: SNSTopic
```
Comment (kharashun) [inline import-service/serverless.yml:56]: lack of filter policy

**Review by ihargit — COMMENTED** [review]
some things might be improved:
- no possibility to test from front-end
- filter policy implemented not at full extend
- part of task functionality might be moved to product service

File: import-service/serverless.yml (hunk 3 above)
Comment (Emerkof) [reply import-service/serverless.yml:20]: Agree

Please, consider hiding sensitive data (emails/passwords/etc) into `.env` files

File: import-service/src/catalogBatchProcess.js [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+        `select p.id, p.description, p.price, p.title, s.count from products p left join stocks s on s.product_id = p.id where p.id = $1`,
+        [newId]
+      );
+    });
+
+    sns.publish(
+      {
+        Subject: "Email from amazon",
+        Message: "New products created",
+        TopicArn: process.env.SNS_ARN,
```
Comment (Emerkof) [inline import-service/src/catalogBatchProcess.js:52]: To make your filter policy work, you should specify `MessageAttributes`, as described [here](https://docs.aws.amazon.com/sns/latest/dg/sns-subscription-filter-policies.html)

I'm giving you 5/6, but please make sure to fully implement the SNS filter policy

**Review by Emerkof — CHANGES_REQUESTED** [review]

**Pull request merged**